core/src/tab/tab.tsx

```diff
@@ -21,57 +21,56 @@ interface TabStyle {
 
 interface Props {
 	children: Tab[];
+	activeTab: string;
+	setActiveTab: (tab: string) => void;
```

Comment:
These should not be required. We should support both controlled and uncontrolled use case